### PR TITLE
Updated trp_test filename

### PR DESCRIPTION
**Pull request by pdeguzman96**

### CODE REVIEW

File: app/web/models/orders.py [hunk 1]
```diff
@@ -101,7 +101,7 @@ def format_date(self):
         except:
             current_app.logger.warning("Invoice date was not picked.")
 
-    def set_order_values(self, page_obj, template_name = 'sysco.json'):
+    def set_order_values(self, page_obj, template_name):
```
Comment (PansulBhatt) [inline app/web/models/orders.py:104]: You sure you want to remove this? Its a good check

File: app/web/preprocessor/trp_process_doc.py [hunk 2]
```diff
@@ -39,26 +39,6 @@ def set_orderitem(self, orderitem_buf, orderitem_tsv, orderitem_obj = None):
 
     def processDocument(self):
         for page in self._raw_doc.pages:
-            # print("PAGE\n====================")
```
Comment (PansulBhatt) [inline app/web/preprocessor/trp_process_doc.py:42]: Thanks for clearing this out

File: app/web/models/orders.py (hunk 1 above)
Comment (pdeguzman96) [reply app/web/models/orders.py:104]: Oh I removed it for the demo. I can add it back but it shouldn't ever make a difference

Comment (pdeguzman96) [reply app/web/models/orders.py]: Just added it back. I noticed one of the tests failed because it relies on that being set as a default

**Pull request merged**